apps/server/src/modules/provisioning/strategy/sanis/response/sanis-geburt-response.ts

```diff
@@ -0,0 +1,3 @@
+export interface SanisGeburtResponse {
+	datum?: string;
```

Comment:
This is an extra object because from sanis we also can get geburtsort for example